src/chapters/setup-intro-data-types/more-data-types.rst

```diff
@@ -26,19 +75,19 @@ using the ``new`` keyword:
 
 .. index:: ! reference type
 
-Just like variables can be declared as an ``int`` or ``String``
-data type, they can also be declared as a specific *class* type. Variables
-that are class type---as opposed to primitive types like ``double`` or object
-types like ``String``---are said to be of **reference type**. Using this
-terminology, ``myCat`` is a *reference variable* of type ``Cat``.
+Just like a variable can be declared as a primitive data type like ``char`` or
+``double``, it can also be declared as a specific *class* type. Variables
+that hold objects---like ``String`` or ``Cat``---are said to be of
```

Comment:
i suggest saying, 'like any ``String`` variable or the object ``Cat``', to distinguish between that data type and a user defined type

Reply:
Revised as per our longer reference type conversation.